tests/e2e/volume_snapshot_test.go

```diff
@@ -240,6 +241,10 @@ var _ = Describe("Verify Volume Snapshot",
 					// Create a "test" table with values 1,2
 					AssertCreateTestData(namespace, clusterToSnapshotName, tableName, psqlClientPod)
 
+					// Because GetCurrentTimestamp() rounds down to the second and is executed
+					// right after the creation of the test data, we wait for 1s to avoid not
+					// including the newly created data within the recovery_target_time
+					time.Sleep(1 * time.Second)
 					// Get the recovery_target_time and pass it to the template engine
```

Comment:
could we make use of a more precise format such as `YYYY-MM-DD HH24:MI:SS.FF6TZH` in `GetCurrentTimestamp` function?

Reply:
This is a good suggestion, especially as the PITR argument allows for microseconds.
Going to use `US` rather than `FF6`, as `FF6` is not supported until Postgres 13

Reply:
`to_char(current_timestamp, 'YYYY-MM-DD HH24:MI:SS.USTZH')`
Tested with EPAS and Postgres.

Reply:
Keeping the buffer fix.